tests/cephadm/test_admin_node_removal.py

```diff
@@ -0,0 +1,33 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify removal of _admin node acting as a bootstrap node
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83574670
+    Test Case Flow:
+    1. Deploy a cluster and check for admin node
+    2. Try to remove the _admin node from bootstrap node
+    """
+
+    try:
+        config = kw.get("config")
+
+        # Removing _admin node from bootstrp node
+        instance = CephAdmin(cluster=ceph_cluster, **config)
+        admin_node = instance.cluster.get_nodes()[0].hostname
+        out, err = instance.installer.exec_command(
+            cmd="cephadm shell ceph orch host rm " + admin_node, sudo=True
+        )
+        return 1
```

Comment:
Missing validation for removed host

Reply:
Thanks @vamahaja

As per the test case , Removing "Admin node" not allowed and above command should though error.
As result we have added "Exception" to remove command